docs/repositories-coverage/commits.md

```diff
@@ -76,3 +76,19 @@ The option **Show also files without coverage changes** allows you to list all f
 ![Files tab](images/{{ page.meta.file_name }}-tab-files.png)
 <!-- vale on -->
 <!--tab-files-end-->
+
+<!--tab-diff-start-->
+## Diff tab
+
+The **Diff** tab displays a line-by-line view of the coverage variation introduced by the {{ page.meta.page_name }}. It includes the following areas:
+
+-   A list of files modified by the {{ page.meta.page_name }}, showing next to each file either the total file coverage or the coverage variation introduced by the {{ page.meta.page_name }}.
+
+-   A diff viewer comparing the old and new content of each modified file.
+
+    Any added or modified coverable lines are highlighted in **red** if not covered and in **green** if covered. Each covered line includes a label showing its test coverage count.
```

Comment:
Here we could follow [an already used format](https://docs.codacy.com/repositories/pull-requests/#diff-tab) to explain each color, using bullets. For example, it would be clearer to have the details for covered lines on the "Green" bullet. ("Each covered line includes a label showing its test coverage count.")

What do you think?

Reply:
Good idea. Will do 👍